MakefileVC

```diff
@@ -36,7 +36,7 @@ else
 	VC6_LINK = wine $(VC6_BIN_DIR)/link.exe
 endif
 
-CFLAGS=/nologo /c /TC /GX /W3 /O1 /I $(VC6_INC_DIR) /FD /MT /D "NDEBUG" /D "WIN32" /D "_WINDOWS" /Fp"Diablo.pch" /YX /Gm /Zi /FA
+CFLAGS=/nologo /c /TC /GX /W3 /O1 /I $(VC6_INC_DIR) /FD /MT /D "NDEBUG" /D "WIN32" /D "_WINDOWS" /Fp"Diablo.pch" /YX /Gm /Zi /FAs

```

Comment:
Was this change intended for this commit? What does `/FA` do as compared to `/FAs`?

Reply:
yeah, it not only shows the lines in asm file but the code too, so you don't have to switch between VS and asm file to find the lines with the code, imo very handy

Reply:
AFAIK /FA generates .asm files with line number for all compiled files. Where /FAs generates .asm files that are interlaced with source. I'm ok with the change.

Reply:
/FA
; Line 4283
	mov	esi, ecx
	push	edi
	imul	esi, 176				; 000000b0H
	push	5

Reply:
/FAs
; 4267 : {

	push	ebp
	mov	ebp, esp
	sub	esp, 60					; 0000003cH
	push	ebx
	push	esi

; 4268 : 	int v1;           // esi
; 4269 : 	int *v2;          // eax
; 4270 : 	int v4;           // ecx
; 4271 : 	int v5;           // ebx
; 4272 : 	int v6;           // ecx
; 4273 : 	int v7;           // edx
; 4274 : 	int v8;           // ecx
; 4275 : 	int v9;           // ST10_4
; 4276 : 	int v10;          // ecx
; 4277 : 	int ExpLight[14]; // [esp+Ch] [ebp-3Ch]
; 4278 : 	int ia;           // [esp+44h] [ebp-4h]
; 4279 : 
; 4280 : 	v1 = i;
; 4281 : 	ExpLight[3] = 5;
; 4282 : 	ExpLight[4] = 5;
; 4283 : 	missile[v1]._miyoff += 32;